open-company/open-company-web
# Ux mobile: navbar and navigation refresh

Card: https://trello.com/c/CmdtEAKW

To test:
- on mobile
- [x] sections navigation looks like design? Good
- go to AP
- [x] navbar looks good? Good
- go to MS
- [x] navbar looks good? Good
- go to a section
- [x] navbar looks good? Good
- go to drafts
- [ ] navbar looks good? Good
- now click on the search icon
- [x] does it look like design? Does it work? Good
- now click on the notifications bell
- [x] does it look like design? Good
- now open navbar
- click on the user icon on the right
- [x] menu looks like design? Good
- click my profile
- [x] navbar ok? Good
- from user menu click notifications settings
- [x] navbar ok? Good
- from user menu click reminders
- [x] navbar ok? Good
- [x] check the following views on desktop for regression: navbar, user menu, user and notifications settings, org settings (team, user management and invite), reminders, notifications, search
